proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -28,6 +28,9 @@ service DocService {
   // GetSourceFile retrieves the source contents for the given owner, repository,
   // reference, and path.
   rpc GetSourceFile(GetSourceFileRequest) returns (GetSourceFileResponse);
+  // GetModulePackages retrieves the list of packages for module based on the given
+  // owner, repository, and reference.
+  rpc GetModulePackages(GetModulePackagesRequest) returns (GetModulePackagesResponse);
   // GetModuleDocumentation retrieves the documentation for module based on the given
   // owner, repository, and reference.
   rpc GetModuleDocumentation(GetModuleDocumentationRequest) returns (GetModuleDocumentationResponse);
```

Comment:
I like the separation in this way, my only concern is that we need to make sure that we are changing the frontend calls at the same time (and make sure that we deploy the frontend changes at the same time), since we are changing the nature of an existing endpoint and also making a breaking change to the field.

I think I would be more comfortable if this is an incremental change: support the package list for both endpoints first, and then migrate the change to remove the package list... and also, I don't think we need to change the tag number from `3->2` for the documentation... we can simply remove the old `repeated packages = 2;` field... but this is just my opinion.